src/OpenTK/Platform/Windows/API.cs

```diff
@@ -135,7 +135,7 @@ internal static class Functions
         );
 
         [DllImport("shell32.dll")]
-        internal static extern uint DragQueryFile(
```

Comment:
You could change DllImport to DllImport("shell32.dll", charset=CharSet.Unicode), would allow you to keep the "nice" name DragQueryFile.